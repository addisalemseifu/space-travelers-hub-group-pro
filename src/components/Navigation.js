import React from 'react';
import { Link, NavLink } from 'react-router-dom';
import planet from './planet.png';

export default function Navigation() {
  return (
    <nav>
      <img src={planet} alt="" />
      <li className="nav-item">
        <NavLink
          to="/"
          className={({ isActive }) => (isActive ? 'active' : '')}
        >
          {' '}
          Dragon
        </NavLink>
      </li>
      <Link to="/profile">
        <li className="navlink" title="profile">
          My Profile
        </li>
      </Link>
    </nav>
  );
}
