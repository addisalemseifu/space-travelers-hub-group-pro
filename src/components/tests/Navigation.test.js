import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Navigation from '../../components/Navigation';

it('renders correctly', () => {
  const navbar = renderer
    .create(
      <Router>
        <Navigation />
      </Router>,
    )
    .toJSON();
  expect(navbar).toMatchSnapshot();
});