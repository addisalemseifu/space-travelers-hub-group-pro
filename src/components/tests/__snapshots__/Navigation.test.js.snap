// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav>
  <div
    className="logoname"
  >
    <img
      alt=""
      src="planet.png"
    />
    <h2>
      Space Travelers' Hub
    </h2>
  </div>
  <div
    className="navcont"
  >
    <li
      className="links"
    >
      <a
        ClassName="link"
        NavLink={true}
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        Rockets
      </a>
    </li>
    <li
      className="nav-item"
    >
      <a
        className=""
        href="/dragons"
        onClick={[Function]}
      >
         
        Dragon
      </a>
    </li>
    <li>
      <a
        ClassName="mission"
        className=""
        href="/mission"
        onClick={[Function]}
      >
        Missions
      </a>
    </li>
    <li
      className="nav-item"
    >
      <a
        className=""
        href="/profile"
        onClick={[Function]}
      >
         
        My Profile
      </a>
    </li>
  </div>
</nav>
`;
